tests/queries/0_stateless/02428_parameterized_view.sql

```diff
@@ -0,0 +1,57 @@
+DROP TABLE IF EXISTS v1;
+DROP TABLE IF EXISTS v2;
+DROP TABLE IF EXISTS v3;
+DROP TABLE IF EXISTS Catalog;
+DROP TABLE IF EXISTS system.v1;
+DROP TABLE IF EXISTS system.Catalog;
+
+CREATE TABLE Catalog (Name String, Price UInt64, Quantity UInt64) ENGINE = Memory;
+
+INSERT INTO Catalog VALUES ('Pen', 10, 3);
+INSERT INTO Catalog VALUES ('Book', 50, 2);
+INSERT INTO Catalog VALUES ('Paper', 20, 1);
+
+CREATE VIEW v1 AS SELECT * FROM Catalog WHERE Price={price:UInt64};
```

Comment:
btw what if in a create view query there are query parameters which are partially substituted by clickhouse client / http parameters? Add a test?

Reply:
I added a test for with client parameters, they will not be directly used, but can be propagated. 
Eg:
set param_price=10;
SELECT * from view1(price={price:UInt64}). -> works

SELECT * from view1; -> gives UNKNOWN_QUERY_PARAMETER error